lstchain/scripts/lstchain_data_muon_analysis_dl1.py

```diff
@@ -69,7 +74,12 @@ def main():
                          'impact_x_array': [],
                          'impact_y_array': [],
                          }
-    
+
+    calib.read_file(args.calib_file)
+    bad_pixels = calib.calib_data.unusable_pixels[0]
+    print("Found a total of", np.sum(bad_pixels), "pixels.")
+    good_pixels = np.logical_not(bad_pixels)
```

Comment:
Just as a minor comment (can be ignored):
- you can write `print(f"Found a total of {np.sum(bad_pixels)} pixels.")`
- `good_pixels = ~bad_pixels` is equivalent. This way you can avoid defining `good_pixels` and later just do `image = full_image*(~bad_pixels)`

Reply:
Done, thx